sw/host/opentitanlib/src/io/jtag.rs

```diff
@@ -46,14 +46,16 @@ pub enum JtagError {
 impl_serializable_error!(JtagError);
 
 /// A trait which represents a JTAG interface.
```

Comment:
Would you mind adding a little more docs on what a JTAG chain is and why it's different to the JTAG trait?